templates/main/multipassage.html

```diff
@@ -114,7 +126,7 @@ <h3 class="entry-title text-center">{{object.collections.current.title}}<a id="{
             {%set all_reffs = objects|map(attribute='subreference')|list %}
             {%set new_reffs = objects|map(attribute='subreference')|list %}
             {% set navigation = nav(text, all_texts, new_reffs, loop.index0) %}
-            {% if current_user.project_team == False and 'andecavensis' in text.objectId %}
+            {% if current_user.project_team == False %}
```

Comment:
The check of whether 'andecavensis' is in the objectId needs to stay. Why did you delete it in the first place?